lib/kafka/offset_manager.rb

```diff
@@ -111,17 +116,51 @@ def seconds_since_last_commit
       Time.now - @last_commit
     end
 
+    def seconds_since_last_recommit
+      Time.now - @last_recommit
```

Comment:
Do we have to handle `@last_recommit` being nil? I don't think it will happen currently due to a `@last_recommit.nil?` check in the calling site (`recommit_timeout_reached?`) but it might be worth coding this more defensively.